src/libraries/System.Security.Cryptography.Algorithms/ref/System.Security.Cryptography.Algorithms.cs

```diff
@@ -544,6 +544,7 @@ public abstract partial class RandomNumberGenerator : System.IDisposable
         public static void Fill(System.Span<byte> data) { }
         public abstract void GetBytes(byte[] data);
         public virtual void GetBytes(byte[] data, int offset, int count) { }
+        public static byte[] GetBytes(int byteCount) { throw null; }
```

Comment:
Minor, but just want to check: @GrabYourPitchforks, this argument name could just be `count` to match the `count` on the other GetBytes overload... did we consider that?  I'm ok either way.

Reply:
TBH I don't have a strong stance on what the arg name should be. If _count_ flows better then so be it. :)

Reply:
I am hearing I should change it then?

Reply:
The only other `GetBytes(int)` that I'm seeing (other than BitConverter.GetBytes, which has a very different meaning) calls the parameter `cb`, which is not to be emulated.

If someone used named syntax then `byte[] random = RandomNumberGenerator.GetBytes(count: 32);` looks more fluid than `byte[] random = RandomNumberGenerator.GetBytes(byteCount: 32);`... so I guess I support simplifying the name to `count`.